From 6006da7a5d78f878c95dfbb48800a2a99c8e6f27 Mon Sep 17 00:00:00 2001
From: Ikey Doherty <ikey@solus-project.com>
Date: Wed, 28 Feb 2018 16:10:00 +0000
Subject: [PATCH 4/4] interfaces/built-in: Remove core-breaking line

With advice from zyga on IRC, we discovered this was the culprit for
breaking the installation of core when this interface is active.

Signed-off-by: Ikey Doherty <ikey@solus-project.com>
---
 interfaces/builtin/steam_support.go | 1 -
 1 file changed, 1 deletion(-)

diff --git a/interfaces/builtin/steam_support.go b/interfaces/builtin/steam_support.go
index 369eea661..3b9ee4d6c 100644
--- a/interfaces/builtin/steam_support.go
+++ b/interfaces/builtin/steam_support.go
@@ -23,7 +23,6 @@ const steamSupportSummary = `allows access to various permissions required to us
 
 const steamSupportBaseDeclarationSlots = `
   steam-support:
-    allow-installation: false
     deny-connection: true
     deny-auto-connection: true
 `
-- 
2.16.2

